### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Uthmanbello**
## Highlights
- Style the website
- Add needed icons
- Fix errors using linters

### CODE REVIEW

File: src/components/Book.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
+.list-container {
```
Comment (MeqdamAlqudah) [inline src/components/Book.css:1]: - [x] Please there should be a line between the update progress button and the progress bar as in the design 
Zeplin
image

Your design 
image

Comment (MeqdamAlqudah) [inline src/components/Book.css:1]: - [x] Please add distance between the books cards and the right side of the screen and between the add book form and the bottom of the screen as in the design 
your design 
image
Zeplin
image

Comment (MeqdamAlqudah) [inline src/components/Book.css:1]: - [x] Please as in Zeplin's design the width of the add book form section and the books cards section should be the same, please adjust the width of add book section and the width of the input fields to match the design.
You design
image
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Uthmanbello , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Highlights
:+1: no linter errors 
:+1: nice styling 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- fixing the overflow of your project it should be more user friendly

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.css (hunk 1 above)
Comment (Uthmanbello) [reply src/components/Book.css:1]: @MeqdamAlqudah,
Thanks for the review. However, this is not how the page displays on my Google Chrome. It does not overflow to the right in my browser.

Thanks

File: src/index.css [hunk 2]
```diff
@@ -1,19 +1,11 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat:wght@700&display=swap"');
+
 body {
```
Comment (Ismailco) [inline src/index.css:3]: - [ ] [OPTIONAL] Kindly make sure to reduce the space at the right of the book's card so the left and right white space are the same and the book cards are centered

Screenshot_20230304_231417

**Review by Ismailco — APPROVED** [review]
### Hi @Uthmanbello, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**